homeworks/src/homework3/part2/Animal.java

```diff
@@ -0,0 +1,40 @@
+package homework3.part2;
+
+public class Animal {
+    public static int animalsCounter;
+
+    public String name;
+
+    public Animal() {
+        animalsCounter += 1;
+    }
+
+    public int getAnimalsCounter(){
+        return animalsCounter;
+    }
+
+    public void run(String name, int distance){
+        System.out.println(name + " пробежал " + distance + " м");
+    }
```

Comment:
the name field is redundant, it is coming from Animal class

Reply:
got it! fixed.